ouroboros-consensus/src/consensus-testlib/Test/Ouroboros/Consensus/ChainGenerator/Honest.hs

```diff
@@ -292,7 +292,11 @@ would solve the problem with just two toggles.
 uniformTheHonestChain ::
   forall base hon g.
      R.RandomGen g
-  => Maybe Asc   -- ^ 'Nothing' means @0@, which induces a periodic chain
+  => Maybe Asc   -- ^ When @Nothing@, the generated schema has a minimal amount
+                 -- of active slots. Deactivating any of them would violate
+                 -- safety properties. The generated schema shows a periodic
+                 -- structure, but this is more an artifact of the implementation
+                 -- than a feature.
```

Comment:
This sounds a bit like that there might be non-periodic minimally dense schemas, but the implementation is just not generating them. Maybe
```suggestion
                 -- safety properties. The generated schema shows a periodic
                 -- structure which is determined by the first stability window.
```